src/container/container_iv.c

```diff
@@ -787,6 +792,7 @@ struct iv_capa_ult_arg {
 	uuid_t		pool_uuid;
 	uuid_t		cont_uuid;
 	uuid_t		cont_hdl_uuid;
+	bool		invalid_current;
```

Comment:
Shortening "invalidate" to "invalid" is potentially confusing for a boolean field.